addons/crm/tests/test_crm_lead_cancel.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+
+from openerp.addons.crm.tests.test_crm_access_group_users import TestCrmAccessGroupUsers
+
+class TestCrmLeadCancel(TestCrmAccessGroupUsers):
+
+    def test_crm_lead_cancel(self):
+        """ Tests for Crm Lead Cancel """
+        CrmTeam = self.env['crm.team']
+        crmlead = self.env.ref("crm.crm_case_1")
+
+        # I set a new sale team giving access rights of salesman.
```

Comment:
comment not match with below code ;) giving access rights of salesman and give salesmanager rights ... ;)